### DISCUSSION TITLE
eslint react-native/no-unused-styles with `createStylesheet` from unistyles

### DISCUSSION THREAD

**Discussion by bryanprimus** [jpudysz/react-native-unistyles#184] [Q&A] [answered]
[eslint-plugin-react-native](https://www.npmjs.com/package/eslint-plugin-react-native) has useful rule to detect whether styles being used or not.

however this isn't work with `createStylesheet` from `unistyles`

we can add this settings in eslint config to other stylesheet providers.
```tsx
settings: {
    'react-native/style-sheet-object-names': ['createStylesheet']
}
```

but it seems like it only works if we use `Stylesheet` from `react-native`.

can we make this rule works with `unistyles`?

**Comment by jpudysz** [comment, marked as answer]
Hey 👋 

It was already discussed here multiple times. This currently has low priority for me. I prefer to work on core rather than building and maintaining the tools. 

I'm looking for contributions from the community ☺️
